just_audio/lib/just_audio.dart

```diff
@@ -3772,9 +3771,9 @@ class AndroidEqualizer extends AudioEffect with AndroidAudioEffect {
     }
     final response = await platform
         .androidEqualizerGetParameters(AndroidEqualizerGetParametersRequest());
-    _parameters =
+    final parameters =
```

Comment:
This local variable shadows the getter of the same name and causes an error on line 3769: "Local variable 'parameters' can't be referenced before it is declared".

You might either rename the local variable to something like `receivedParameters` or change line 3769 to use `this.parameters`.

Reply:
Yikes sorry for the noise. Thanks for the suggestion. Applied.